wordpress-mobile/WordPress-iOS
# Pasting images crashes the editor.

Fixes #6751

To test:
 - Start a new post using Aztec
 - Switch to the Photos app
 - Copy an image on the Photos app
 - Switch to the WP app
 - Paste the image
 - Check if paste worked and upload happens correctly
 - Switch to html and see if the URL is correct.

Needs review: @diegoreymendez
